recipes/cunit/all/conanfile.py

```diff
@@ -57,89 +63,79 @@ def _settings_build(self):
         return getattr(self, "settings_build", self.settings)
 
     def build_requirements(self):
-        self.build_requires("libtool/2.4.6")
-        if self._settings_build.os == "Windows" and not tools.get_env("CONAN_BASH_PATH"):
-            self.build_requires("msys2/cci.latest")
+        self.build_requires("libtool/2.4.7")
+        if self._settings_build.os == "Windows":
+            self.win_bash = True
+            self.build_requires("autoconf/2.71")
+            self.tool_requires("automake/1.16.5")  # Needed for complie and lib wrappers
```

Comment:
Never, it doesn't scale (it's what we were doing in the past and it was a nightmare of versions conflicts).

Reply:
@SpaceIm I tried to remove the `automake` dependency, but then the build fails on windows.

Check the generate method, it uses `self.conf.get("user.automake:compile-wrapper")` and this one is None, if automake dependency is not given...

Reply:
Yes, it seems that `self.conf.get()` can't see conf coming from transitive dependencies.
But you can use `self.dependencies.build["automake"].conf_info.get("user.automake:compile-wrapper", check_type=str)` instead.